tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -521,6 +521,13 @@ def fit(self,
                 data. e.g. 500 hand-labeled samples (perhaps a hard human task), whole data set (unlabeled) is thousands/millions.
                 However, this isn't the only use case. Given enough unlabeled data(millions of rows), you may see improvements
                 to any amount of labeled data.
+            weight_column : str, default = None
```

Comment:
It might make sense for this to actually live in init:

i.e.:

```
predictor = TabularPredictor(label='class', sample_weight='__weight__', weight_evaluation=True, eval_metric='log_loss').fit(train_data)
```

Additionally, these parameters can be sent to Learner init instead of Learner fit.